Genres/filter.js

```diff
@@ -0,0 +1,30 @@
+const filterBar = async (arg) => {
+  const url = constructUrl(`movie/${arg}`);
+  const res = await fetch(url);
+  const data = await res.json();
+  if (arg !== "latest") {
+    renderMovies(data.results);
+  } else {
+    renderMovie(data);
+  }
+};
```

Comment:
Really like how you made a single function for everything. Keep up the good work 🎉